### PR TITLE
[Fix](security) All sk in ms log should be encrypted

### PR DESCRIPTION
**Pull request by Yukang-Lian**
### What problem does this PR solve?

Issue Number: close #xxx

Related PR: #xxx

Problem Summary:

### Release note

None

### Check List (For Author)

- Test 
    - [ ] Regression test
    - [x] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [x] No.
    - [ ] Yes. 

- Does this need documentation?
    - [x] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by Yukang-Lian** [comment]
run buildall

File: cloud/src/meta-service/meta_service_helper.h [hunk 1]
```diff
@@ -101,7 +116,25 @@ void finish_rpc(std::string_view func_name, brpc::Controller* ctrl, Response* re
...
+            // Found "sk" field, extract its value and perform MD5 encryption
+            size_t sk_value_start = debug_string.find('\"', pos) + 1;
+            size_t sk_value_end = debug_string.find('\"', sk_value_start);
+
+            std::string sk_value =
+                    debug_string.substr(sk_value_start, sk_value_end - sk_value_start);
+            std::string encrypted_sk = md5(sk_value);
+
+            // Replace the original "sk" value with the encrypted MD5 value
+            debug_string.replace(sk_value_start, sk_value_end - sk_value_start, encrypted_sk);
```
Comment (gavinchou) [inline cloud/src/meta-service/meta_service_helper.h:134]: TEST_SYNC_PIONT("sk response", &debug_string);


test.cpp
```
SyncPoint::set_callback("sk response", [](args) {
EXPECT_NE(((string*) args[0])->find("sk: xxxx"), std::string::npos);
}
);
```

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run buildall

**Comment by Yukang-Lian** [comment]
run cloudut

**Comment by Yukang-Lian** [comment]
run cloudut

**Review by gavinchou — APPROVED** [review]

**Review by liaoxin01 — APPROVED** [review]
LGTM

**Pull request merged**